dataclients/kubernetes/kube.go

```diff
@@ -1008,15 +1008,19 @@ func createEastWestRoute(eastWestDomainRegexpPostfix, name, ns string, r *eskip.
 	ewR := *r
 	ewR.HostRegexps = []string{"^" + name + "[.]" + ns + eastWestDomainRegexpPostfix + "$"}
 	ewR.Id = patchRouteID(r.Id)
-	log.Infof("*** ewR: %s: %s", ewR.Id, ewR.String())
 	return &ewR
 }
 
 func createEastWestRoutes(eastWestDomainRegexpPostfix, name, ns string, routes []*eskip.Route) []*eskip.Route {
-	var ewroutes []*eskip.Route = make([]*eskip.Route, 0, len(routes))
+	ewroutes := make([]*eskip.Route, 0)
```

Comment:
this init value is never used, in such cases I find it cleaner to write just `var ewroutes []*eskip.Route`